upgrade/scripts/upgrade/util/update-customizations.sh

```diff
@@ -134,7 +134,16 @@ if [[ "$(yq r "$c" "spec.kubernetes.services.cray-sysmgmt-health.prometheus-snmp
 fi
 
 # Kube-prometheus-stack
-sed -i 's/prometheus-operator/kube-prometheus-stack/g' "$c"
+if [[ "$(yq r "customizations.yaml" "spec.kubernetes.services.cray-sysmgmt-health.prometheus-operator")" ]]; then
+      yq merge -xP -i "$c" <(yq r "$c" spec.kubernetes.services.cray-sysmgmt-health.kube-prometheus-stack | yq p - spec.kubernetes.services.cray-sysmgmt-health.prometheus-operator)
+      yq w -i "$c" "spec.proxiedWebAppExternalHostnames.customerManagement[3]" "{{ kubernetes.services['cray-sysmgmt-health']['kube-prometheus-stack'].prometheus.prometheusSpec.externalAuthority }}"
+      yq w -i "$c" "spec.proxiedWebAppExternalHostnames.customerManagement[4]" "{{ kubernetes.services['cray-sysmgmt-health']['kube-prometheus-stack'].alertmanager.alertmanagerSpec.externalAuthority }}"
+      yq w -i "$c" "spec.proxiedWebAppExternalHostnames.customerManagement[5]" "{{ kubernetes.services['cray-sysmgmt-health']['kube-prometheus-stack'].grafana.externalAuthority }}"
+      yq w -i "$c" "spec.kubernetes.services.cray-kiali.kiali-operator.cr.spec.external_services.grafana.url" "https://{{ kubernetes.services['cray-sysmgmt-health']['kube-prometheus-stack'].grafana.externalAuthority }}/"
+      yq4 eval ".spec.kubernetes.services.cray-sysmgmt-health.kube-prometheus-stack = .spec.kubernetes.services.cray-sysmgmt-health.prometheus-operator | del(.spec.kubernetes.services.cray-sysmgmt-health.prometheus-operator)" -i $c
+      yq w -i "$c" "spec.kubernetes.services.cray-sysmgmt-health.kube-prometheus-stack.prometheus.prometheusSpec.externalUrl" "https://{{ kubernetes.services['cray-sysmgmt-health']['kube-prometheus-stack'].prometheus.prometheusSpec.externalAuthority }}/"
+      yq w -i "$c" "spec.kubernetes.services.cray-sysmgmt-health.kube-prometheus-stack.alertmanager.alertmanagerSpec.externalUrl" "https://{{ kubernetes.services['cray-sysmgmt-health']['kube-prometheus-stack'].alertmanager.alertmanagerSpec.externalAuthority }}/"
```

Comment:
To help us move towards yq version 4, please use the version 4 command and its syntax

```
yq4 -i '.a.b = "value"' "${c}"
```